### PR TITLE
refactoring .env loading

### PR DESCRIPTION
**Pull request by ndefokou**

### CODE REVIEW

File: generic-server/src/main.rs [hunk 1]
```diff
@@ -4,6 +4,13 @@ use std::net::SocketAddr;
 
 #[tokio::main]
 async fn main() {
+
+    // Load dotenv-flow variables
+    dotenv_flow::from_filename(".env.example").ok();
```
Comment (IngridPuppet) [inline generic-server/src/main.rs:9]: The code must not depend on `.env.example`. Each one will have to create a `.env` file on its computer; it is fine. Remember to add `.env` to `.gitignore`.
```suggestion
    dotenv_flow::dotenv_flow().ok();
```

**Review by IngridPuppet — CHANGES_REQUESTED** [review]

File: generic-server/src/main.rs (hunk 1 above)
Comment (IngridPuppet) [reply generic-server/src/main.rs:9]: Hello

Comment (IngridPuppet) [reply generic-server/src/main.rs:9]: There are also conflicts to resolve.

Comment (ndefokou) [reply generic-server/src/main.rs:9]: Okay i am just from correcting it

Comment (IngridPuppet) [reply generic-server/src/main.rs:9]: I still see `.env.example` in the code.

**Comment by IngridPuppet** [comment]
You shouldn't have removed `.env.example` on functions annotated with `#[cfg(test)]`. It is okay to rely on the file for tests.

**Review by IngridPuppet — APPROVED** [review]
Working.

**Pull request merged**